tweak(docs/naitves/LoadResourceFile): Improved the explanation of one of the parameters (#2672)
citizenfx/fivem: ext/native-decls/LoadResourceFile.md

Reads the contents of a text file in a specified resource.
If executed on the client, this file has to be included in `files` in the resource manifest.
Example: `local data = LoadResourceFile("devtools", "client/data.json")`

## Parameters
* **resourceName**: The resource name.

> * **directory**: The path of the file in the resource.

Reviewer: ```suggestion
* **filePath**: The path to the file, relative to the specified resource
```
This would make more sense over directory, the parameters must also match the function declaration
